### ISSUE TITLE
After install, running docker version  reports an error: error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.38/version: open //./pipe/docker_engine: The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect.  This error may also indicate that the cker daemon is not running

### ISSUE THREAD

**Issue by ilevina** [docker/for-win#2367]
- [x ] I have tried with the latest version of my channel (Stable or Edge)
  - [x ] I have uploaded Diagnostics
  - Diagnostics ID: 177BCDFE-555D-4648-A597-373C42CD8E75/20180803170822

### Expected behavior
After install I should run successfully docker version command
### Actual behavior
Executing docker version produce the following:

Client:
 Version:           18.06.0-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        0ffa825
 Built:             Wed Jul 18 19:05:28 2018
 OS/Arch:           windows/amd64
 Experimental:      false
error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.38/version: open //./pipe/docker_engine: The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.

### Information
  - Windows Version: Windows 10 Pro
  - Docker for Windows Version:   18.06.0-ce
Before installing Docker For Windows on my laptop, I verified that the following Windows Features are enabled:
Hyper-V
Containers 
Verified  that Virtualization is enabled in Task Manager. 

### Steps to reproduce the behavior
Problem is reproducible ( at least on windows laptop)
  1. Install Docker For Windows. Install seems successful. At least I could see that Docker Windows Service is running.
  2. However when in PS command I  tried to execute:  docker version it shows error that reported above. Log file reported an error:   [DockerDaemonChecker][Error  ] Docker daemon is not running
3. I tried to restart Docker  Windows Service as suggested in troubleshooting guide. And then trying to restart docker by  double clicking Docker image on Desktop. Docker shows message: Docker is running .
However I am still getting same error  when trying to execute the following commands: 
docker version 
docker ps  or trying to run hello_world

**Comment by mostm** [comment]
Did you run PowerShell as administrator?

**Comment by ilevina** [comment]
Yes, I run PowerShell as administrator. I am getting this error every time on Restart: docker startup tales a long time and then it always fails with error, I reported.   It seems not only me having similar issue with installing Docker on Windows 10 Pro.  I tried several times to re-installed it, always ended with  same result.  

To re-install steps:

1) Uninstall Docker. After un-install delete the following folders manually:  c:/Users/…/AppData/Local/Docker, c:/Users/…/Roaming/Docker/, c:/ProgramData/Docker,c:/ProgramFiles/Docker
2) Disable Hyper-V and Containers, restart computer
3) Enable Hyper-V, restart computer. Enable Containers restart computer. 
4) Then install Docker-For-Windows  and … Nothing help. Same error. 

If you have any idea, what else could be done to install Docker , welcome with suggestions.
Thx

**Comment by sergioSN** [comment]
I have the same problem, at the moment I have not been able to solve it

@ilevina, have you been able to solve it?

**Comment by ilevina** [comment]
No, I didn't solve it

**Comment by sergioSN** [comment]
I've got it working, I've removed all the traces with a program called Geek (https://geekuninstaller.com) and then I've reinstalled

**Comment by Relaxe111** [comment]
I've got same problem:
`C:\>docker version
Client:
 Version:           18.06.1-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        e68fc7a
 Built:             Tue Aug 21 17:21:34 2018
 OS/Arch:           windows/amd64
 Experimental:      false
error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.38/version: open //./pipe/docker_engine: The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.`

**Comment by cicorias** [comment]
this is just happening out of the blue. Next time I see this I'll do a better job of inspecting if the `\\pipe` still is present or defined in the OS. From my situation, all was working great from a non-admin shell.

Then, I see this error which is false. But, using the command to see if the pipe is still presents, idicates it is NOT. 

My fix was just reset Docker via the Applet.

# Error message of no sense 
```
C:\g\cse\mcd\azdo\nlp-pipeline [master ≡ +0 ~2 -0 !]> docker ps
error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.38/containers/json: open //./pipe/docker_engine: The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.
```
# Before a Reset of Docker. 
![image](https://user-images.githubusercontent.com/1080406/46826841-e9969000-cd4b-11e8-8026-5c457120c3ab.png)


# After the Reset:

![image](https://user-images.githubusercontent.com/1080406/46826899-121e8a00-cd4c-11e8-8a77-39cbbbbc8e55.png)

**Comment by FoxNeo** [comment]
![](https://i.ibb.co/MZpFmP6/solution.jpg)

This works
Docker **Reset to factory Defaults**

**Comment by aristenio** [comment]
Docker Reset to factory Defaults didn't orked for me.

**Comment by desireddysahitya** [comment]
> ![](https://camo.githubusercontent.com/6d08ea34dbe5c837dbf95f9e99b17139e43218e8/68747470733a2f2f692e6962622e636f2f4d5a70466d50362f736f6c7574696f6e2e6a7067)
> 
> This works
> Docker **Reset to factory Defaults**

This really worked out Thanks a lot

**Comment by chronos25** [comment]
hi ! I'm facing this error and unable to solve. Can anyone brief it clearly how to solve.
Machine OS - Windows 10 HOME.
After Installing docker toolbox.

```
$ docker info
Client:
 Debug Mode: false

Server:
ERROR: error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.40/info: open //./pipe/docker_engine: The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.
errors pretty printing info
```

**Comment by chowdaryprasaad** [comment]
cd "C:\Program Files\Docker\Docker"
./DockerCli.exe -SwitchDaemon

try this

**Comment by pomil** [comment]
I tried the command 
cd "C:\Program Files\Docker\Docker"
./DockerCli.exe -SwitchDaemon

but then I get following Error when running _**docker run hello-world: **_

Unable to find image 'hello-world1:latest' locally
C:\Program Files\Docker\Docker\Resources\bin\docker.exe: Error response from daemon: Get https://registry-1.docker.io/v2/library/hello-world1/manifests/latest: unauthorized: incorrect username or password.
See 'C:\Program Files\Docker\Docker\Resources\bin\docker.exe run --help'.

**Comment by redhead** [comment]
@chowdaryprasaad 
This worked for me. Just beware you need to switch two times (e.g. first to windows containers, and then again to go back to linux containers).

Thanks.

**Comment by War777** [comment]
> cd "C:\Program Files\Docker\Docker"
> ./DockerCli.exe -SwitchDaemon
> 
> try this

This worked for me, thanks a lot.

**Comment by koziolk** [comment]
> > cd "C:\Program Files\Docker\Docker"
> > ./DockerCli.exe -SwitchDaemon
> > try this
> 
> This worked for me, thanks a lot.

With what version of Docker Windows?

**Comment by eschrader** [comment]
> > cd "C:\Program Files\Docker\Docker"
> > ./DockerCli.exe -SwitchDaemon
> > try this
> 
> This worked for me, thanks a lot.

This worked for me after I had removed the Hyper V feature and network adapters. I reset docker desktop to factory and ran this command once and docker ps worked.

**Comment by Luis-Enrique1** [comment]
i use windows 10, but is not a professional edition so i've used this tutorial 
https://itnext.io/install-docker-on-windows-10-home-d8e621997c1d, 
and i've tried everything, i've installed virtual box, i've installed linux, made it on linux, i've got problems with kubernetes, i've resitalled everything 10 times, disabled and enabled everything, get tons of errors, i have this

error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.40/containers/json: open //./pipe/docker_engine: The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.

i've tried 
bcdedit /set hypervisorlaunchtype auto 
bcdedit /set hypervisorlaunchtype off
docker toolbox
installing a virtual machine
enabling and disabling hyper-v
reisntalling
installing windows pro
in the virtual machine i have an error with kuberntete cluster

i really want to use docker.... i need it, there is a problem with amd cpu?

**Comment by koseavase** [comment]
> i use windows 10, but is not a professional edition so i've used this tutorial
> https://itnext.io/install-docker-on-windows-10-home-d8e621997c1d,
> and i've tried everything, i've installed virtual box, i've installed linux, made it on linux, i've got problems with kubernetes, i've resitalled everything 10 times, disabled and enabled everything, get tons of errors, i have this
> 
> error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.40/containers/json: open //./pipe/docker_engine: The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.
> 
> i've tried
> bcdedit /set hypervisorlaunchtype auto
> bcdedit /set hypervisorlaunchtype off
> docker toolbox
> installing a virtual machine
> enabling and disabling hyper-v
> reisntalling
> installing windows pro
> in the virtual machine i have an error with kuberntete cluster
> 
> i really want to use docker.... i need it, there is a problem with amd cpu?


Hello Luis-Enrique1

Even was facing the same issue, it seems Docker 2.3.3 is not mature and has lot of issues.
Please go back to old version of docker (18.06), I did the same and its working

I am summarizing the steps

1. Open powershell in administrative mode, Copy the below content in a bat file (InstallHyperV.bat)
InstallHyperV.bat
================================================================================
pushd "%~dp0"
dir /b %SystemRoot%\servicing\Packages\*Hyper-V*.mum >hyper-v.txt
for /f %%i in ('findstr /i . hyper-v.txt 2^>nul') do dism /online /norestart /add-package:"%SystemRoot%\servicing\Packages\%%i"
del hyper-v.txt
dism /online /enable-feature /featurename:Microsoft-Hyper-V -All /LimitAccess /ALL
pause
===================================================================================

2. run InstallHyperV.bat, and check if there are not issues

3. open Registry Editor in "Administrative mode"
   go to Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion
   Double click "EditionID" and change the Value date to "Professional"
   Don't close the window

4. Install  docker (18.06)

5. Open powershell in administrative mode, and type docker ps


Hope it should work for you!!!!!

**Comment by rj1711** [comment]
> cd "C:\Program Files\Docker\Docker"
> ./DockerCli.exe -SwitchDaemon
> 
> try this

how to use this in windows 10. as I do not have "Docker" file in C:\Program Files. I have "Docker toolbar" instead.Please help

**Comment by gordonma05** [comment]
I followed the tutorial https://itnext.io/install-docker-on-windows-10-home-d8e621997c1d to install docker in my windows 10 home and got the same issue when I restarted my machine every time. I have to uninstall and install docker multiple times to get it work again. It seems windows uninstall doesn't fully uninstall docker because uninstall is very fast and takes only a couple seconds. Now I have to keep my machine sleep instead of shutting down to avoid reinstalling dockers.
